dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -42,10 +46,58 @@
 public final class GitHubHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(GitHubHelper.class);
+    public static final String BRANCHNAME_FOR_BOT = "feature/add_dockstore_yml";
+
+    public static final String DOCKSTORE_BOT_PR_TEXT = """
+           The dockstore-bot has guessed at a .dockstore.yml for your repository.
+           
+           Please review, make appropriate changes, add any [additional keys](https://docs.dockstore.org/en/stable/assets/templates/template.html) and merge in order to complete integration with Dockstore, allowing Dockstore to keep informed of new changes to your workflow(s), notebook(s), or tool(s).
```

Comment:
Should we mention the Dockstore GitHub App anywhere? I'm unsure of how informed the user is of the GitHub app feature at the time of this PR. If the user initiates the .dockstore.yml inference process, then I suppose they should know things about the GitHub App feature. If another user of the organization comes across the PR, they may not know much about it.

Maybe it wouldn't hurt to add the main [GitHub App documentation link](https://docs.dockstore.org/en/stable/getting-started/github-apps/github-apps-landing-page.html)?